EIPS/eip-beacon_state_root_in_evm.md

```diff
@@ -0,0 +1,169 @@
+---
+eip: <to be assigned>
+title: Beacon state root in the EVM
+description: Expose beacon chain state roots in the EVM
+author: Alex Stokes (@ralexstokes), Danny Ryan (@djrtwo)
+discussions-to: <TODO: eth-magicians URL>
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): Core
+created: 2022-02-10
+---
+
+## Abstract
+
+Commit to the state root of the beacon chain in the `ommers` field in the post-merge execution block. Reflect the changes in the `ommersHash` field of the execution block header.
```

Comment:
Recommend referring to these fields by their position, since the names are meaningless (especially post-merge since it is just a fixed value).  You could say something like "in position `x`, which formerly held the `ommersHash`".

Reply:
Thank you for your suggestions. I have made targeted modifications based on your feedback and have added a file containing detailed information related to the proposal. Thank you very much for your contribution.